client/starwhale/core/dataset/store.py

```diff
@@ -312,16 +322,18 @@ def from_data_link_uri(cls, data_uri: str, auth_name: str) -> ObjectStore:
     def from_dataset_uri(cls, dataset_uri: URI) -> ObjectStore:
         if dataset_uri.object.typ != URIType.DATASET:
             raise NoSupportError(f"{dataset_uri} is not dataset uri")
+        return cls(
+            backend=SWDSBackendType.LocalFS,
+            bucket=str(DatasetStorage(dataset_uri).data_dir.absolute()),
+        )
 
-        _type = dataset_uri.instance_type
-        if _type == InstanceType.STANDALONE:
-            backend = SWDSBackendType.LocalFS
-            bucket = str(DatasetStorage(dataset_uri).data_dir.absolute())
-        else:
-            backend = SWDSBackendType.S3
-            bucket = os.environ.get("SW_S3_BUCKET", _DEFAULT_S3_BUCKET)
-
-        return cls(backend=backend, bucket=bucket)
+    @classmethod
+    def from_remote_dataset(cls, dataset_uri: URI, auth_name: str) -> ObjectStore:
```

Comment:
what's different between [from_dataset_uri](https://github.com/star-whale/starwhale/pull/1443/files#diff-30bbfec45080e5cc83b6569a57f62aedfaffd66e819ce77222896ead95ce6c0dR322) and [from_remote_dataset](https://github.com/star-whale/starwhale/pull/1443/files#diff-30bbfec45080e5cc83b6569a57f62aedfaffd66e819ce77222896ead95ce6c0dR331)?

Reply:
[from_remote_dataset](https://github.com/star-whale/starwhale/pull/1443/files#diff-30bbfec45080e5cc83b6569a57f62aedfaffd66e819ce77222896ead95ce6c0dR331) will build a signedUrlBackend, [from_dataset_uri](https://github.com/star-whale/starwhale/pull/1443/files#diff-30bbfec45080e5cc83b6569a57f62aedfaffd66e819ce77222896ead95ce6c0dR322) will build a boto3.s3 backend